storage/volume/core.go

```diff
@@ -97,13 +119,35 @@ func (c *Core) GetVolume(id types.VolumeID) (*types.Volume, error) {
 		if err = client.New().Get(url, v); err == nil {
 			return v, nil
 		}
-		if ce, ok := err.(client.Error); ok && ce.IsNotFound() {
-			return nil, volerr.ErrVolumeNotFound
-		}
+	}
+
+	// at last, scan all drivers
+	logrus.Debugf("probing all drivers for volume with name: %s", id.Name)
```

Comment:
Please remove `:` or use `logrus.Debugf("probing all drivers for volume with name(%s)", id.Name)`